metrics/telemetry.go

```diff
@@ -169,6 +169,13 @@ var (
 			Name:      "compact_partition_usage",
 			Help:      "Counter of compact table partition",
 		})
+	TelemetryDistReorgCnt = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "telemetry",
+			Name:      "Distributed_reorg_count",
```

Comment:
```suggestion
			Name:      "distributed_reorg_count",
```

metrics's name should be lower case.